apps/web/src/assets/css/web.scss

```
@@ -27,6 +27,30 @@
   border-bottom-color: #8eb8dc;
 }
 
+// Match OG CaTH footer size: govuk-frontend v6 bumped the footer from
+// size 16 to 19 as part of the GDS rebrand. Restore size 16 (16px at all
+// breakpoints) so the footer matches the legacy service.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)